src/meta/expr.lean

```diff
@@ -404,11 +416,11 @@ meta def is_eta_expansion_test : list (name × expr) → option expr
 
 /-- Checks whether there is an expression `e` such that for all *non-propositional* elements
   `(nm, val)` in the list `val = e ... nm` where `...` denotes the same list of parameters for all
-  applications. Also checks whether the resulting expression unifies with the given one -/
-meta def is_eta_expansion_aux (val : expr) (l : list (name × expr)) : tactic (option expr) :=
+  applications. Also checks whether the resulting expression unifies with `given` -/
```

Comment:
I know it isn't a new addition, but I find this doc string incomprehensible. (Your change helps slightly.) If you understand the function, could you rephrase it?

Reply:
better now?

Reply:
Yes, thanks!